### PR TITLE
Updated cluster get to return list of subnets

### PR DESCRIPTION
**Pull request by Praveengostu**

### CODE REVIEW

File: api/container/containerv1/clusters.go [hunk 1]
```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
...
 	WorkerCount       int
+	Vlans             []Vlan
+}
+
+type Vlan struct {
+	ID      string
+	Subnets []SubnetV
+}
+
+type SubnetV struct {
```
Comment (ashishth09) [inline api/container/containerv1/clusters.go]: Can we call it just Subnet?

File: api/container/containerv1/clusters.go [hunk 2]
```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
...
+	CIDR     string
```
Comment (ashishth09) [inline api/container/containerv1/clusters.go]: How about just Cidr, Id, Ips?

File: api/container/containerv1/clusters.go [hunk 3]
```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
 	State             string
 	IsPaid            bool
 	WorkerCount       int
+	Vlans             []Vlan
+}
+
+type Vlan struct {
+	ID      string
```
Comment (ashishth09) [inline api/container/containerv1/clusters.go]: How about just Id?

Comment (Praveengostu) [reply api/container/containerv1/clusters.go]: In all the cluster structs we follow ID so used the same

File: api/container/containerv1/clusters.go (hunk 1 above)
Comment (Praveengostu) [reply api/container/containerv1/clusters.go]: Subnet is already used . So added V denoting Vlan

File: api/container/containerv1/clusters.go (hunk 2 above)
Comment (Praveengostu) [reply api/container/containerv1/clusters.go]: Id puting it as ID . Changed Cidr and Ips

File: api/container/containerv1/clusters_test.go [hunk 4]
```diff
@@ -215,7 +215,18 @@ var _ = Describe("Clusters", func() {
...
+				"Subnets": [
+				{
+				"Cidr": "159.8.226.208/29",
+				"ID": "1541737",
+				"Ips": ["159.8.226.210"],
+				"IsByOIP": false,
+				"IsPublic": false
+				}]
+				}]}`),
 					),
```
Comment (ashishth09) [inline api/container/containerv1/clusters_test.go]: How about asserting the values in the test?

Comment (Praveengostu) [reply api/container/containerv1/clusters_test.go]: Did that

**Pull request merged**